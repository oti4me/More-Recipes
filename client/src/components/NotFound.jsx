import React from 'react';
import Header from './Header'
import Footer from './Footer'

/**
 * @description Renders the not found page
 * 
 * @param {object} props
 * 
 * @returns {object} jsx object that renders the not found page 
 */
export const NotFound = (props) => (
  <div className="notFoundDiv main">
    <Header />
    <div className="cont">
      <h1 className="notFoundText" style={{ textAlign: "center", paddingTop: "50px" }}>Not found</h1>
    </div>
    <Footer />
  </div>
)

export default NotFound;