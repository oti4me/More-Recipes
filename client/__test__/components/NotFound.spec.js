import React from 'react';
import NotFound from '../../src/components/NotFound';

describe('Routes', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});