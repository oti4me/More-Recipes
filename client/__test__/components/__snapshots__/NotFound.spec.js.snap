// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(Header) />,
        <div
          className="cont"
        >
          <h1
            className="notFoundText"
            style={
              Object {
                "paddingTop": "50px",
                "textAlign": "center",
              }
            }
          >
            Not found
          </h1>
        </div>,
        <Footer />,
      ],
      "className": "notFoundDiv main",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="notFoundText"
            style={
              Object {
                "paddingTop": "50px",
                "textAlign": "center",
              }
            }
          >
            Not found
          </h1>,
          "className": "cont",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Not found",
            "className": "notFoundText",
            "style": Object {
              "paddingTop": "50px",
              "textAlign": "center",
            },
          },
          "ref": null,
          "rendered": "Not found",
          "type": "h1",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(Header) />,
          <div
            className="cont"
          >
            <h1
              className="notFoundText"
              style={
                Object {
                  "paddingTop": "50px",
                  "textAlign": "center",
                }
              }
            >
              Not found
            </h1>
          </div>,
          <Footer />,
        ],
        "className": "notFoundDiv main",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="notFoundText"
              style={
                Object {
                  "paddingTop": "50px",
                  "textAlign": "center",
                }
              }
            >
              Not found
            </h1>,
            "className": "cont",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Not found",
              "className": "notFoundText",
              "style": Object {
                "paddingTop": "50px",
                "textAlign": "center",
              },
            },
            "ref": null,
            "rendered": "Not found",
            "type": "h1",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
